op-node/rollup/driver/state.go

```diff
@@ -228,7 +246,15 @@ func (s *Driver) eventLoop() {
 				// This may adjust at any time based on fork-choice changes or previous errors.
 				//
 				// update sequencer time if the head changed
```

Comment:
These comments are for `planSequencerAction`. Please move these comments down to `planSequencerAction`.